internal/cmd/envs.go

```diff
@@ -482,6 +483,19 @@ coder envs edit back-end-env --disk 20`,
 				return err
 			}
 
+			if !force && env.LatestStat.ContainerStatus == coder.EnvironmentOn {
+				_, err = (&promptui.Prompt{
+					Label:     fmt.Sprintf("Rebuild environment \"%s\"? (will destroy any work outside of /home)", env.Name),
```

Comment:
it doesn't matter for anything, but fyi you can use `%q` to double quote your string (you can tell that I am extremely lazy)

```suggestion
					Label:     fmt.Sprintf("Rebuild environment %q? (will destroy any work outside of /home)", env.Name),
```

You might also want to mention "your home directory" instead of a specific path, because as a special case, root's home directory is `/root`

Reply:
I agree on "your home directory"

Reply:
Oh nice - didn't know about that `%q` 🤯 🙌 

And yeah the rename from `/home` to `your home directory` is a good call. Fixed it on the rebuild prompt too 👌